portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/dependencies/com.liferay.portal.security.sso.openid.connect.internal.configuration.OpenIdConnectProviderConfiguration~default.config

```diff
@@ -4,8 +4,8 @@ discoveryEndPointCacheInMillis=L"360000"
 idTokenSigningAlgValues=["RS256"]
 issuerURL="http://127.0.0.1:18080/auth/realms/master"
 jwksURI="http://127.0.0.1:18080/auth/realms/master/protocol/openid-connect/certs"
-openIdConnectClientId="liferay-system-level"
-openIdConnectClientSecret="449c61a0-69b5-4202-8470-202b187a9414"
+openIdConnectClientId="${openid.provider.default.client.id}"
+openIdConnectClientSecret="${openid.provider.default.client.secret}"
```

Comment:
Should be openid.provider.default.client.password: https://github.com/liferay/liferay-jenkins-ee/blob/master/commands/build.properties#L1101